core/src/main/java/com/graphhopper/util/PathSimplification.java

```diff
@@ -98,6 +98,16 @@ public PointList simplify() {
                     if (removed > 0) {
                         for (int i = 0; i < toSimplify.size(); i++) {
                             reduceLength(toSimplify.get(i), offset[i], startIntervals[i], endIntervals[i] - removed);
+                            /*
```

Comment:
But this is probably not really necessary as a first version (and probably even later)